### ISSUE TITLE
VPAT — Label Errors In Discussion Forum

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#3976]
The Discussion forum has many errors and warnings noted by WAVE.  This issue will focus on the "Missing Form Label" errors.

Screenshot:
![Screen Shot 2019-07-01 at 5 38 39 PM](https://user-images.githubusercontent.com/12703624/60468661-54fcf680-9c28-11e9-816c-89c187dbaf86.png)

Please add the `<label>` tag to form controls.  Any controls that are not visually labeled can, instead, be labeled by an `aria-label` (or similar) property.

About `<label>`: https://www.w3schools.com/tags/tag_label.asp
About `aria-label`: https://webaim.org/techniques/forms/advanced
